Sources/HeliumLogger/HeliumLogHandler.swift

```diff
@@ -0,0 +1,111 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Logging
+
+public extension HeliumLogger {
+    
+    /// Creates a `HeliumLogHandler` instance for use with the SwiftLog logging system.
+    ///
+    ///### Usage Example: ###
+    /// This may be used to bootstrap the SwiftLog logging system with a
+    /// `HeliumLogger` instance. The `HeliumLogger` instance will
+    /// be used as the logging backend for SwiftLog.
+    ///```swift
+    ///let heliumLogger = HeliumLogger()
+    ///LoggingSystem.bootstrap(heliumLogger.makeLogHandler)
+    ///```
+    /// - Parameter label: The label to use for a SwiftLog `Logger`.
+    func makeLogHandler(label: String) -> HeliumLogHandler {
+        return HeliumLogHandler(label: label, logger: self)
+    }
+    
+    private static let defaultLogger = HeliumLogger()
+    
+    /// Creates a `HeliumLogHandler` instance for use with the SwiftLog logging system.
+    ///
+    ///### Usage Example: ###
+    /// This may be used to bootstrap the SwiftLog logging system with a default
+    /// `HeliumLogger` instance. The default `HeliumLogger` instance will
+    /// be used as the logging backend for SwiftLog.
+    ///```swift
+    ///LoggingSystem.bootstrap(HeliumLogger.makeLogHandler)
+    ///```
+    /// - Parameter label: The label to use for a SwiftLog `Logger`.
+    static func makeLogHandler(label: String) -> HeliumLogHandler {
+        return HeliumLogHandler(label: label, logger: defaultLogger)
+    }
+}
+
+/// A lightweight implementation of SwiftLog's `LogHandler` protocol.
+public struct HeliumLogHandler: LogHandler {
+    public var logger: HeliumLogger
+    
+    public let label: String
+    
+    public var logLevel: Logger.Level = .info
+    
+    private var prettyMetadata: String?
+    
+    public var metadata = Logger.Metadata() {
+        didSet {
+            prettyMetadata = prettify(metadata)
+        }
+    }
+    
+    public subscript(metadataKey metadataKey: String) -> Logger.Metadata.Value? {
+        get {
+            return metadata[metadataKey]
+        }
+        set {
+            metadata[metadataKey] = newValue
+        }
+    }
+    
+    init(label: String, logger: HeliumLogger) {
+        self.label = label
+        self.logger = logger
+    }
+    
+    public func log(level: Logger.Level, message: Logger.Message, metadata: Logger.Metadata?, file: String, function: String, line: UInt) {
+        let prettyMetadata = metadata?.isEmpty ?? true
+            ? self.prettyMetadata
+            : self.prettify(self.metadata.merging(metadata!, uniquingKeysWith: { _, new in new }))
+        
```

Comment:
It may be worth noting that this pretty metadata logic comes from SwiftLog's [built in logger](https://github.com/apple/swift-log/blob/b33865f66bc32553ff5366506f3cad1f704f0a13/Sources/Logging/Logging.swift#L580).

I was tempted to change it a bit (it feels a little dense to parse the ternary / nil coalescing combo), but noticed that Ian has the same code in [LoggingSyslog](https://github.com/ianpartridge/swift-log-syslog/blob/master/Sources/LoggingSyslog/SyslogLogHandler.swift#L20), too. 

I figured keeping it obviously the same was better than functionally the same / syntactically different.